sdk/identity/identity-cache-persistence/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "@azure/identity-cache-persistence",
+  "version": "1.0.0-beta.1",
+  "description": "A secure, persistent token cache for Azure Identity credentials that uses the OS secret-management API",
+  "main": "dist/index.js",
+  "module": "dist-esm/identity-cache-persistence/src/index.js",
+  "types": "./types/identity-cache-persistence.d.ts",
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:samples": "echo skipped",
+    "build:test": "tsc -p . && rollup -c rollup.config.js 2>&1",
+    "build": "npm run extract-api && tsc -p . && rollup -c 2>&1",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\" \"samples-dev/**/*.ts\"",
+    "clean": "rimraf dist dist-* typings *.tgz *.log",
+    "execute:samples": "echo skipped",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\" \"samples-dev/**/*.ts\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test  --ext .ts",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "mocha -r esm -r ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace --exclude \"test/**/browser/**/*.spec.ts\" \"test/**/*.spec.ts\"",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/identity/src",
+    "dist-esm/identity-cache-persistence/src",
+    "types/identity-cache-persistence.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "repository": "github:Azure/azure-sdk-for-js",
+  "keywords": [
+    "azure",
+    "cloud",
+    "active directory",
+    "authentication",
+    "credential",
+    "certificate",
+    "managed service identity",
+    "client secret",
+    "access token"
```

Comment:
should we put keywords specifically related to persistence?

Reply:
Good catch. This was copied from mainline identity, so I'll add some keywords both to this and to the vscode package.